Task/src/luiscachi/movies/Main.java

```diff
@@ -0,0 +1,10 @@
+package luiscachi.movies;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Since now you are using unit tests, this class is no longer necessary.
Please remove it!.